imperial_coldfront_plugin/views.py

```diff
@@ -264,3 +264,61 @@ def get_active_users(request: HttpRequest) -> HttpResponse:
         )
 
     return HttpResponse(passwd)
+
+
+@login_required
+def make_group_manager(request: HttpRequest, group_membership_pk: int) -> HttpResponse:
+    """Make a group member a manager.
+
+    Args:
+        request: The HTTP request object containing metadata about the request.
+        group_membership_pk: The primary key of the group membership to be updated.
+    """
+    group_membership = get_object_or_404(GroupMembership, pk=group_membership_pk)
+    group = group_membership.group
+
+    if (
+        request.user != group.owner
+        and not request.user.is_superuser
+        and not GroupMembership.objects.filter(
+            group=group, member=request.user, is_manager=True
+        ).exists()
```

Comment:
It's a fair question whether managers should be able to make other managers but I'd be tempted to lean on the side of no.